tests/suites/test_suite_mpi.function

```diff
@@ -1720,6 +1720,378 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE */
+void mbedtls_mpi_core_add_if( char * input_l, char * input_r,
+                              char * input_X4, int carry4,
+                              char * input_X8, int carry8 )
+{
+    mbedtls_mpi X4, X8, l, r;
+    mbedtls_mpi_uint *la = NULL;
```

Comment:
I get that `l` and `r` are left and right operands. Except that they actually aren't used as left and right operands. `X` is the expected result, which is not at all transparent but is a common convention in this file — I'd still prefer `expected_X` or `expected_result` or `expected_sum`. I have no idea what `d` stands for.

Please either use meaningful names (preferred), or follow the old convention of A/B/… for inputs and X/Y/Z for outputs.